scripts/release.sh

```diff
@@ -509,6 +509,7 @@ function main() {
     # HACK HACK HACK
     # Rerun the release script from the release branch. Fixes https://github.com/knative/test-infra/issues/1262
     ./hack/release.sh "$@"
+    exit "$?"
```

Comment:
are you sure stop here?

Reply:
Yes, at this point, the logic below would have been executed in the `./hack/release.sh` in the release branch, so we should stop here.